Client/Frontend/Shields/SimpleShieldsView.swift

```diff
@@ -110,11 +158,16 @@ class SimpleShieldsView: UIView, Themeable {
         }
         
         func applyTheme(_ theme: Theme) {
+            let contentBackgroundColor = theme.isDark ? UIColor(rgb: 0x303443) : Colors.neutral000
+            descriptionStackView.addBackground(color: contentBackgroundColor, cornerRadius: 6.0)
+            infoStackView.addBackground(color: contentBackgroundColor, cornerRadius: 6.0)
+            shareStackView.addBackground(color: contentBackgroundColor, cornerRadius: 6.0)
+            
             countLabel.textColor = theme.isDark ? .white : .black
-            descriptionLabel.textColor = theme.isDark ? UIColor.white : .black
-            infoButton.tintColor = theme.isDark ?
-                Colors.orange400 :
-                Colors.orange500
+            descriptionLabel.textColor = theme.isDark ? .white : .black
+
+            infoButton.tintColor = theme.isDark ? .white : .black
+            shareButton.tintColor = theme.isDark ? . white : .black
```

Comment:
I think you may need to use `appearanceTintColor` (and possibly `appearanceTextColor` in other places) for iOS 12 theming support

Reply:
iOS 12 theming support along side with accessibility elements are added